docs/ContinuousIntegration.md

```diff
@@ -0,0 +1,117 @@
+# Firebase Apple SDK Continuous Integration
+
+Firebase uses [several GitHub Action workflows](.github/actions/) for product testing. Most of the
+tests are presubmit tests that are triggered by a PR making changes to any of a
```

Comment:
It might be nice to start off with a description of the tests like. What do you think about something like:

```
There are two types of tests: 
1. Presubmit tests: triggered by a PR when changing specific files
2. Nightly tests: run once a day, scheduled overnight. Potentially longer-running tests.

Presubmit tests are also scheduled to run overnight, independent of PRs. 
```